modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/util/ServiceContextUtil.java

```diff
@@ -106,4 +121,73 @@ private static boolean _isObjectEntryDraft(Status status) {
 		return false;
 	}
 
+	private static void _updateTaxonomyCategoryIds(
```

Comment:
@AliciaGarciaGarcia just rename the method name to `_setTaxonomyCategoryIds`.

Thanks!